src/extensions/default/InlineColorEditor/css/main.less

```diff
@@ -457,3 +457,5 @@
         }
     }
 }
+
+@sansFontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
```

Comment:
This is what Brackets does, we're just removing the web font.  I'd rather file a new bug afterward to evaluate our font choices as part of a larger UX overhaul in Thimble vs. doing it in this PR.